__tests__/api.test.js

```diff
@@ -52,16 +54,38 @@ describe("API tests", ()=>{
                 expect(response.body).toHaveProperty("votes", 100)
             })
         })
-        test("responds with 400 status code when given invalid article_id endpoint", () => {
+        test("400: Responds with error when given invalid article_id endpoint", () => {
             return request(app)
             .get("/api/articles/banana")
             .expect(400)
             .then((response) => {
                 expect({msg:"Article ID must be a number"})
             })
         })
-    }
-
-    )
-})
-})
+    })
+    describe("GET /api/articles", () => {
+        test("200: Responds with an articles array of article objects containing info and comment count in descending order", ()=>{
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response) => {
+                expect(Array.isArray(response.body))
+                response.body.forEach((article) => {
+                expect(article).toHaveProperty("author", expect.any(String))
+                expect(article).toHaveProperty("comment_count", expect.any(Number))
+                })
+            })
+        })
+        test("200: Check there isn't a body property", ()=>{
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response) => {
+                response.body.forEach((article) => {
+                expect(article).not.toHaveProperty("body")
+                })
+            })
+        })
+    })
+    describe("", () => {}) 
```

Comment:
You need to have a status 200 test that checks the default sort & order is correct -` created_at, desc`
Have a look at the `jest-sorted` library to help you do this